governance.md

```diff
@@ -1,133 +1,233 @@
+
+### Member
+
+Members are continuously active contributors in the community. They can have issues and PRs assigned to them. Members are expected to remain active contributors to the community.
+
+**Defined by**: Member of the Github organization.
+
+#### Requirements
+
+* Enabled [two-factor authentication](https://help.github.com/articles/about-two-factor-authentication) on their GitHub account.
+* Have read the [Contributor Guide](https://spinnaker.io/community/contributing/).
+* Have made multiple contributions to the project or community. Contributions may include, but is not limited to:
+  * Authoring or reviewing PRs on GitHub
+  * Filing or commenting on Issues in GitHub
+  * Contributing to SIG or community discussions (e.g. meetings, Slack, email discussions)
+* Subscribed to [spinnaker-dev@googlegroups.com](TODO)
+* Sponsored by 2 Reviewers. **Note the following requirements for sponsors**:
+  * Sponsors must have close interactions with the prospective member - e.g. code/design/proposal review, coordinating issues, etc.
+  * Sponsors must be Reviewers or Approvers in any project in the [Spinnaker org](https://github.com/spinnaker).
+  * Sponsors must be from multiple member companies to demonstrate integration across the community.
+* **[Open an issue](https://github.com/spinnaker/governance/issues/new) against the spinnaker/governance repo**
+  * Ensure your sponsors are @mentioned on the issue.
+  * Complete every item on the checklist.
+  * Make sure that the list of contributions included are representative of your work on the project.
+* Have your sponsoring reviewers reply confirmation of sponsorship: `:+1:`.
+* Once your sponsors have responded, your request will be reviewed by the TOC. Any missing information will be requested.
+
+#### Responsibilities and Privileges
+
+* Responsive to issues and PRs assigned to them.
+* Responsive to mentions of SIG teams they are members of.
+* Active owner of code they have contributed (unless ownership is explicitly transferred).
+  * Code is well tested.
+  * Tests consistently pass.
+  * Addresses bugs or issues discovered after code is accepted.
+* They can be assigned to issues and PRs, and people can ask members for reviews.
+
+**Note**: Members who frequently contribute code are expected to proactively perform code reviews and work towards becoming a Reviewer for the project(s) they are active in.
+
+
+### Reviewer
+
+Reviewers are able to review code for quality and correctness on some part of the project. They are knowledgable about both the codebase and software engineering principles. Reviewership is required before becoming an Approver (except under specific conditions; see Approver).
+
+**Defined by**: _reviewers_ entry in the [membership.yml](membership.yml) file.
 
-Duties
+Reviewer status is scoped to a part of the codebase.
```

Comment:
I was thinking that `membership.yml` in the governance repo would go away, but I'm happy to change this PR to be more aligned with using `membership.yml`.

I added the "defined by" stuff because it was part of the k8s stuff, but I liked that there was a clear definition of where people can find where the source of truth is for this information. We can link to it some other way, though.